app/domain/authentication/security/extract_request_remote_addr_info.rb

```diff
@@ -0,0 +1,51 @@
+require 'command_class'
+
+module Authentication
+  module Security
+    ExtractRequestRemoteAddrInfo = CommandClass.new(
+      dependencies: {
+        # InputValidation
```

Comment:
what's the dependency here? when will it happen?

Reply:
Duplicate comment.